src/ucs/sys/string.c

```diff
@@ -81,6 +81,11 @@ void ucs_fill_filename_template(const char *tmpl, char *buf, size_t max)
             pf = pp + 2;
             p += strlen(p);
             break;
+        case 'i':
+            snprintf(p, end - p, "%u", geteuid());
+            pf = pp + 2;
+            p += strlen(p);
```

Comment:
seems we could remove all `p += strlen(p);` in the switch cases and rely on line 95 that does the same?

Reply:
Yes, true.
In fact this function is not used at all by the VFS, so I just improved it as we discussed with Yossi. If we keep these changes, I'll apply the change you suggested